### DISCUSSION TITLE
Welcome to mulysa Discussions!

### DISCUSSION THREAD

**Discussion by tswfi** [TampereHacklab/mulysa#298] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akusav** [comment]
Hi,

My name is Aku a.k.a. ATS_ and @ats_:hacklab.fi in matrix. I am member of Tampere Hacklab community. Mostly I am answering community members questions related to Tampere Hacklab membership, payments and so on. Other free time I am using by improving Tampere Hacklab and making some cool projects related to whatever comes to my mind.

Feel free to ask here, discourse or other community channels. 

BR,
Aku ATS_
